config/navier_stokes_config.yaml

```diff
@@ -66,7 +66,7 @@ default: &DEFAULT
     n_tests: [2000, 1000] #, 1000]
     test_resolutions: [128, 1024] #, 1024] 
     test_batch_sizes: [16, 4] #, 1]
-    positional_encoding: True
+    positional_encoding: False
```

Comment:
Why was this changed to False?

Reply:
Because it happens in the model by default

Reply:
As I mentioned in the main comments, then let's remove that option alltogether -- there should be only one way to do it